### PR TITLE
Multi-cloud clarification

### PR DESCRIPTION
**Pull request by jcharcalla**
In a recent ticket there was some confusion around using both the helm values and cloud-integration.yaml. This is an attemt to clarify that.

https://kubecost.zendesk.com/agent/tickets/2702

### CODE REVIEW

File: aws-cloud-integrations.md [hunk 1]
```diff
@@ -328,15 +328,17 @@ Download template files from the URLs provided below and upload them as the stac
...
 <details>
 	<summary>Attach via ServiceKey And Kubernetes Secret</summary>
 <ul>
 <li><p>Navigate to <a href="https://console.aws.amazon.com/iam">https://console.aws.amazon.com/iam</a> Access Management &gt; Users. Find the Kubecost User and select Security Credentials &gt; Create Access Key. Note the Access key ID and Secret access key. You&rsquo;ll use it to either Create a secret from helm values or Create and use an existing secret.</p>
 	<details>
 		<summary>Create a secret from Helm values</summary>
 
-<p><details>
-	<summary>Create a secret from Helm values</summary></p>
+<details>
```
Comment (thomasvn) [inline aws-cloud-integrations.md:340]: When Github renders this Markdown, I see two dropdowns that say "Create a secret from Helm values". Was there old text that got deleted?

File: aws-cloud-integrations.md [hunk 2]
```diff
@@ -328,15 +328,17 @@ Download template files from the URLs provided below and upload them as the stac
 ### Step 4: Attaching IAM permissions to Kubecost
 Now that the policies have been created, attach those policies to Kubecost. We support the following methods:
 
+> **NOTE:** If you are using the alternative [multi-cloud integration](https://docs.kubecost.com/install-and-configure/advanced-configuration/cloud-integration/multi-cloud) method steps 4 and 5 are not required. The use of "Attach via Pod Annotation on EKS" authentication is optional.
```
Comment (thomasvn) [inline aws-cloud-integrations.md:331]: Small detail. A comma between "method" and "steps" may increase readability?

File: gcp-out-of-cluster.md [hunk 3]
```diff
@@ -17,6 +17,9 @@ GCP users should create [detailed billing export](https://cloud.google.com/billi
 
 ## Step 2:  Visit Kubecost setup page and provide configuration info
 
+
+> **NOTE:** If you are using the alternative [multi-cloud integration](https://docs.kubecost.com/install-and-configure/advanced-configuration/cloud-integration/multi-cloud) method **step 2** is not required.
```
Comment (thomasvn) [inline gcp-out-of-cluster.md:21]: Small detail. A comma between "method" and "steps" may increase readability?

File: aws-cloud-integrations.md (hunk 1 above)
Comment (jcharcalla) [reply aws-cloud-integrations.md:340]: That "Create a secret from Helm values" section is not showing up on the gitbooks page, this was an attempt to fix that. 

I could have made it worse.

https://docs.kubecost.com/install-and-configure/advanced-configuration/cloud-integration/aws-cloud-integrations#step-4-attaching-iam-permissions-to-kubecost

File: aws-cloud-integrations.md (hunk 2 above)
Comment (jcharcalla) [reply aws-cloud-integrations.md:331]: Thanks, I will add them.

File: gcp-out-of-cluster.md (hunk 3 above)
Comment (jcharcalla) [reply gcp-out-of-cluster.md]: done

File: aws-cloud-integrations.md (hunk 1 above)
Comment (jcharcalla) [reply aws-cloud-integrations.md:340]: Yep, that made it worse in GH... It appears on the docs page the sub bullets are not rendering at all. I will revert this change here. Seems like we may need a larger check of the whole docs site to see is issues are elsewhere.

Comment (jcharcalla) [reply aws-cloud-integrations.md]: I reverted it back, but it still shows twice because of how it was orriginally.

**Review by rossfisherkc — APPROVED** [review]
LGTM

**Pull request merged**